README.md

```diff
@@ -273,15 +257,15 @@ I would like to thank...
 
 - **[Question_2]**
 
-  - [Answer_2]
+  - [Answer_2] -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./LICENSE)licensed.
 
```

Comment:
- Hey, the license link is currently broken. please, let's fix this issue to make the readme more professional.

Reply:
@zieeco I have added a link to the license